src/panels/lovelace/elements/hui-state-icon-element.ts

```diff
@@ -0,0 +1,65 @@
+import { html, LitElement } from "@polymer/lit-element";
+
+import "../../../components/entity/state-badge.js";
+
+import { computeTooltip } from "../../../common/string/compute-tooltip";

```

Comment:
Compute tooltip has to be moved inside lovelace elements folder. It is super lovelace specific, nothing common about it.

Reply:
I'll note that for a future PR